misc-tools/fix_permissions.in

```diff
@@ -9,6 +9,12 @@ WEBAPPDIR="@domserver_webappdir@"
 DOMJUDGE_USER="@DOMJUDGE_USER@"
 WEBSERVER_GROUP="@WEBSERVER_GROUP@"
 
+# If there's no DOMJUDGE_USER (because this server is only a webserver)
+# just set the ownership to root instead
+if ! id $DOMJUDGE_USER >/dev/null 2>/dev/null ; then
+    DOMJUDGE_USER="root"
+fi
+
```

Comment:
The "is only a webserver" confuses me a bit. The focus of this script is a machine being a domserver (not judgehost), right. The question is why we do this in the first place. I think so you can run console commands (db migrate, etc). Do you want to break that?